manifests/base/cluster-role.yaml

```diff
@@ -83,6 +83,14 @@ rules:
       - user.openshift.io
     resources:
       - groups
+  - verbs:
+      - get
+      - list
+      - watch
+    apiGroups:
+      - user.openshift.io
+    resources:
+      - users
```

Comment:
Not sure we will need this unless I get a clear answer on the `username` front that makes sense.

Reply:
replied above